supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -164,6 +164,53 @@
 
 *See also*:
 
+[discrete]
+[[operator]]
+==== image:images/yes.png[yes] Operator (noun)
+*Description*: In the context of Kubernetes, an Operator is a method of packaging, deploying, and managing a
+Kubernetes application. A Kubernetes application is an application that is both
+deployed on a Kubernetes cluster (including OpenShift clusters) and managed
+using the Kubernetes APIs and `kubectl` or `oc` tooling.
+
+The term "Operator" in the context of Kubernetes is always capitalized to distinguish it from other types of operators, such as human or mathematical operators. For example:
+
+.Example: Kubernetes Operator
+----
+= Support policy for unmanaged Operators
+
+Individual Operators have a `managementState` parameter in their configuration.
+----
+
+.Example: Mathematical operator
+----
+The following operators and operands are supported in Drools Rule Language:
+
+* + (addition)
+* - (subtraction)
+...
+----
+
+An Operator's full name must be a proper noun, with each word initially
+capitalized. If it includes a product name, defer the product's capitalization
+style guidelines. For example:
+
+- Red Hat OpenShift Logging Operator
+- Prometheus Operator
+- etcd Operator
+- Node Tuning Operator
+- Cluster Version Operator
+
+While "containerized" is allowed, do not use "Operatorize" to refer to building
```

Comment:
```suggestion
Although "containerized" is allowed, do not use "Operatorize" to refer to building
```
Suggesting this change because the ISG says don't use 'while' to mean 'though' or 'although'

Reply:
Thanks, done.